developer_docs/translations.md

```diff
@@ -53,7 +53,7 @@ export function languageKeyToDateLocale(lang) {
 
 ## Translations
 
-* Every component should introduce its own subset of keys inside a dictionary named after the component. 
+* Every component should introduce its subset of keys inside a dictionary named after the component. 
```

Comment:
I think the "own" here helps provide emphasis, but could also be replaced with a different word!